pinot-common/src/main/proto/worker.proto

```diff
@@ -25,7 +25,7 @@ service PinotQueryWorker {
   // Dispatch a QueryRequest to a PinotQueryWorker
   rpc Submit(QueryRequest) returns (QueryResponse);
 
-  rpc SubmitTimeSeries(TimeSeriesQueryRequest) returns (TimeSeriesResponse);
+  rpc SubmitTimeSeries(TimeSeriesQueryRequest) returns (stream TimeSeriesResponse);
```

Comment:
ideally we should not rely on query server for query response, but we'll fix it as part of MSE integration which is coming up soon.